src/transformers/models/conditional_detr/modeling_conditional_detr.py

```diff
@@ -2093,13 +2094,13 @@ def __init__(self, dim, fpn_dims, context_dim):
         self.lay1 = nn.Conv2d(dim, dim, 3, padding=1)
         self.gn1 = nn.GroupNorm(8, dim)
         self.lay2 = nn.Conv2d(dim, inter_dims[1], 3, padding=1)
-        self.gn2 = nn.GroupNorm(8, inter_dims[1])
+        self.gn2 = nn.GroupNorm(min(8, inter_dims[1]), inter_dims[1])
```

Comment:
Same here, this is to allow for a smaller hidden size since GroupNorm wants arg1 >= arg2.